extensions/objects/templates/math-expression-content-editor.component.ts

```diff
@@ -80,6 +80,9 @@ export class MathExpressionContentEditorComponent implements OnInit {
       this.value.mathExpressionSvgIsBeingProcessed = true;
     }
     this.valueChanged.emit(this.value);
+    if (this.value && this.value.raw_latex) {
+      this.localValue.label = this.value.raw_latex || '';
+    }
```

Comment:
We don't need `|| ''` here since we already have an if condition for `this.value.raw_latex` on line 83.

Reply:
Done!